sass/overrides/_modal.scss

```diff
@@ -11,3 +11,7 @@ modal {
         pointer-events: none;
     }
 }
+
+.tooltip-visible-el {
```

Comment:
I think this may be better moved to the pattern library, but if you remove this change from here,  I will approve the PR since it works so much better now 🎉 thank you so much for this fix

Reply:
👍